addons/point_of_sale/static/src/app/screens/ticket_screen/ticket_screen.xml

```diff
@@ -20,7 +20,7 @@
                                 <t t-esc="preset.name"/>
                             </button>
                         </div>
-                        <div class="item d-flex align-items-center justify-content-end">
+                        <div class="item d-flex align-items-center justify-content-end" t-on-click="onClickPageNbr">
```

Comment:
Dont put this here, otherwise when clicking on the buttons, it will display the popup. Or maybe you can stop the click propagation, I let you check